change/react-native-windows-2020-01-24-13-55-58-MS_NoSharedItemsPCH.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "prerelease",
+  "comment": "Removed pch.h from shared items projects",
+  "packageName": "react-native-windows",
+  "email": "vmorozov@microsoft.com",
```

Comment:
>vmorozov [](start = 12, length = 8)

Should this be jshost?

Reply:
No, it is an e-mail of a person who does changes.

---
In reply to: [370869851](https://github.com/microsoft/react-native-windows/pull/3958#discussion_r370869851) [](ancestors = 370869851)